game/script.js

```diff
@@ -4,21 +4,21 @@ window.onkeydown = function (event) {
   }
 };
 
-const map = document.querySelector('.tablecontainer');
+const map = document.querySelector('.table--container');
 
 class Player {
   constructor() {
     this.element = null;
   }
 
   move(where) {
-    if (!where || where.classList.contains("map-wall")) {
+    if (!where || where.classList.contains("map--wall")) {
       return;
     }
 
     this.element.classList.remove('player');
     where.classList.add("player");
-    aplle();
+    apple();
```

Comment:
funckje jabłko() nie mówi za wiele o tym co robi :)

Reply:
Karol's function, I agree it can be redesigned to something like this:
changeLife () {
//some way to add life after placing player on the apple item
//some way to loose life after placing player in vicinity of a monster
}
To be redesigned later, I hope ;)